packages/twenty-front/src/modules/workflow/components/RightDrawerWorkflowSelectActionContent.tsx

```diff
@@ -24,36 +14,25 @@ const StyledActionListContainer = styled.div`
   padding-inline: ${({ theme }) => theme.spacing(2)};
 `;
 
-export const TAB_LIST_COMPONENT_ID =
-  'workflow-select-action-page-right-tab-list';
-
 export const RightDrawerWorkflowSelectActionContent = ({
   workflow,
 }: {
-  workflow: Workflow;
+  workflow: WorkflowWithCurrentVersion;
 }) => {
-  const tabListId = `${TAB_LIST_COMPONENT_ID}`;
-
-  const { tabs, options, handleActionClick } =
-    useRightDrawerWorkflowSelectAction({ tabListId, workflow });
+  const { createStep } = useRightDrawerWorkflowSelectActionCreateStep({
+    workflow,
+  });
 
   return (
     <>
-      <StyledTabListContainer>
-        <TabList loading={false} tabListId={tabListId} tabs={tabs} />
-      </StyledTabListContainer>
-
       <StyledActionListContainer>
-        {options.map((option) => (
-          <MenuItem
-            key={option.id}
-            LeftIcon={option.icon}
-            text={option.name}
-            onClick={() => {
-              handleActionClick(option.id);
-            }}
-          />
-        ))}
+        <MenuItem
```

Comment:
should be a map on all available actions, coming from an util.

Reply:
Done